src/bicep/modules/policyAssignment.bicep

```diff
@@ -0,0 +1,42 @@
+targetScope = 'resourceGroup'
+
+param builtInAssignment string = ''
+param logAnalyticsWorkspaceName string
+param workspaceResourceGroupName string
```

Comment:
for consistency and clarity let's name this `logAnalyticsWorkspaceResourceGroupName`